docs/asciidoc/modules/ROOT/pages/background-operations/triggers.adoc

```diff
@@ -283,24 +309,26 @@ To solve this, we can use `{phase:'afterAsync'}`
 
 .Pause trigger
 
-We have the possibility to pause a trigger without remove it, if we will need it in the future
+We have the possibility to pause a trigger without remove it, if we will need it in the future.
+Note that this procedure and the `apoc.trigger.start` one are eventually consistent,
+so we need to wait a time defined by the configuration `apoc.trigger.refresh`.
```

Comment:
```suggestion
We have the possibility of pausing a trigger without removing it, in case we might need it in the future.
Note that this procedure and the `apoc.trigger.start` ones are eventually consistent,
so we need to wait some amount of time for the changes to propagate. This amount of time is defined by the configuration `apoc.trigger.refresh`.
```